plugins/woocommerce/src/Admin/Features/Settings.php

```diff
@@ -61,22 +61,44 @@ public function enqueue_settings_editor_scripts() {
 		// Make sure the Settings Editor package is loaded.
 		wp_enqueue_script( 'wc-settings-editor' );
 
-		$script_handle          = 'wc-admin-edit-settings';
-		$script_path_name       = 'wp-admin-scripts/settings';
-		$script_assets_filename = WCAdminAssets::get_script_asset_filename( 'wp-admin-scripts', 'settings' );
-		$script_assets          = require WC_ADMIN_ABSPATH . WC_ADMIN_DIST_JS_FOLDER . 'wp-admin-scripts/' . $script_assets_filename;
-		$script_version         = WCAdminAssets::get_file_version( 'js', $script_assets['version'] );
-
-		wp_register_script(
-			$script_handle,
-			WCAdminAssets::get_url( $script_path_name, 'js' ),
-			$script_assets['dependencies'],
-			$script_version,
+		$script_name            = 'wc-admin-edit-settings';
+		$script_path_name       = 'settings';
+		$script_assets_filename = WCAdminAssets::get_script_asset_filename( $script_path_name, 'index' );
+		$script_assets          = require WC_ADMIN_ABSPATH . WC_ADMIN_DIST_JS_FOLDER . $script_path_name . '/' . $script_assets_filename;
+		$style_assets_filename  = WCAdminAssets::get_script_asset_filename( $script_path_name, 'style' );
+		$style_assets           = require WC_ADMIN_ABSPATH . WC_ADMIN_DIST_JS_FOLDER . $script_path_name . '/' . $style_assets_filename;
+
+		wp_enqueue_script(
+			$script_name,
+			WCAdminAssets::get_url( $script_path_name . '/index', 'js' ),
+			array_merge( array( 'wp-edit-site' ), $script_assets['dependencies'] ),
+			WCAdminAssets::get_file_version( 'js', $script_assets['version'] ),
 			true
 		);
```

Comment:
I switched this to a normal script, not wpAdminScript for easier inclusion of styles. I choose not to reuse `WCAdminAssets::register_script` because it automatically adds `WC_ADMIN_APP` as a dependency - although I could change that function to work around that.